volatility3/framework/plugins/linux/sockstat.py

```diff
@@ -0,0 +1,505 @@
+# This file is Copyright 2021 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+# Author: Gustavo Moreira
```

Comment:
We don't tend to include authors to avoid people getting territorial over changes to files they "own" because they originally wrote them.  We all tend to use git commits to show ownership (which is one of the reasons we use merge requests to keep the authorship information intact).  Would you be ok taking this out for consistency with the rest of the codebase please?